tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
Hmm... Could it be, that pytest together with the runner.invoke method, that is used in the unit tests, doesn't handle stderr-results? However, I tried to do a 

    logging.basicConfig(stream=sys.stdout)

before the test, but that still doesn't work. I guess, because pytest interrupts and captures all output. The code you linked isn't inside this pytest/runner.invoke-cage.